console_backend/src/tabs/solution_tab/solution_position_tab.rs

```diff
@@ -424,7 +419,7 @@ impl SolutionPositionTab {
 
     /// Handle PosLLH / PosLLHDepA messages.
     ///
-    /// TODO(johnmichael.burke@) https://swift-nav.atlassian.net/browse/CPP-95
+    /// TODO(johnmichael.burke@) <https://swift-nav.atlassian.net/browse/CPP-95>
```

Comment:
the TODO is also marked done. deleting it as well

Reply:
i got distracted by some rustdoc linting issues, but i turns out this isn't quite done even though it is marked as done in JIRA... leaving it for now.